src/gradletestrunnersmoketest.sh

```diff
@@ -6,24 +6,24 @@ gradletestrunnersmoketest () {
   echo "Building Data Repo client library"
   ../gradlew clean assemble  
   cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-clienttests
-  export ORG_GRADLE_PROJECT_datarepoclientjar=$(ls ../datarepo-client/build/libs/*jar)
-  echo "ORG_GRADLE_PROJECT_datarepoclientjar = ${ORG_GRADLE_PROJECT_datarepoclientjar}"
 
   echo "Running spotless and spotbugs"
   ./gradlew spotlessCheck
   ./gradlew spotbugsMain
 
   echo "Setting Test Runner environment variables"
+  export ORG_GRADLE_PROJECT_datarepoclientjar=$(ls ../datarepo-client/build/libs/*jar)
   export TEST_RUNNER_SERVER_SPECIFICATION_FILE="${NAMESPACEINUSE}.json"
   export TEST_RUNNER_KEY_DIRECTORY_PATH="${GITHUB_WORKSPACE}"
+  echo "ORG_GRADLE_PROJECT_datarepoclientjar = ${ORG_GRADLE_PROJECT_datarepoclientjar}"
   echo "TEST_RUNNER_SERVER_SPECIFICATION_FILE = ${TEST_RUNNER_SERVER_SPECIFICATION_FILE}"
   echo "TEST_RUNNER_KEY_DIRECTORY_PATH = ${TEST_RUNNER_KEY_DIRECTORY_PATH}"
 
   echo "Running test suite"
   ./gradlew runTest --args="suites/PRSmokeTests.json tmp/TestRunnerResults"
 
```

Comment:
if the runTest fails for some reason, will an empty dir get uploaded, or will earlier results bleed in?
I assume we only ever run these once, so that's impossible?

Reply:
If runTest fails, then the action will skip the rest of the step, so nothing would be uploaded. Note that "fails" does not mean that >1 user journey thread throws an exception. It means that the test runner itself threw an exception, so presumably we have a bug in the test runner or something.

I'm not sure what you mean by only run this once, could you maybe restate? These bash commands get run on every PR, same as our integration/connected tests now.